test/python3/libdnf5/comps/test_comps_iterators.py

```diff
@@ -24,8 +24,10 @@
 class TestCompsIterators(base_test_case.BaseTestCase):
     def test_group_query_iterable(self):
         query = libdnf5.comps.GroupQuery(self.base)
+        self.assertIsInstance(query.get_base().get(), libdnf5.base.Base)
         _ = [grp.get_groupid() for grp in query]
```

Comment:
I wanted to test `get_base()` on the `Group`/`Environment` classes, as well, but this test is broken and doesn't actually contain anything in the query.